R/debug.R

```diff
@@ -0,0 +1,35 @@
+##' Get debug output from naomi
+##'
+##' @title Get debug output from naomi
+##'
+##' @param id The model run id to download. This will be printed at
+##'   the top of the stack trace.
+##'
+##' @param server The url of the server.  The default is to use the
+##'   production naomi/hint/hintr instance. It is not possible to use
+##'   the staging instance.  You can change this if running locally.
+##'
+##' @param dest The destination for the downloaded data.  The actual
+##'   data will be unpacked into a directory corresponding to the run
+##'   id within this, so it is safe to use a common directory.
+##'
+##' @export
+download_debug <- function(id, server = NULL, dest = tempfile()) {
```

Comment:
This will mean that anyone with the model run ID will be able to retrieve the input data from the workshops? Is the security that they have know the model run ID enough?

Reply:
yes - chance of guessing the ID is close to zero, though we are passing it around over http rather than https